### PR TITLE
Feat/star

### PR DESCRIPTION
**Pull request by ivyhu630**
Main change is the implementation of Star which can be used in other components. 

FYI I tried to use style components originally (see commented out codes in Star) but couldn't make it work. Let me know if you know a fix. Current styling for star is pure css.

### CODE REVIEW

File: client/src/Star.jsx [hunk 1]
```diff
@@ -1,14 +1,68 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
+import React from 'react';
+import PropTypes from 'prop-types';
+// import styled from 'styled-components';
 
-export default function Star() {
+export default function Star({ score }) {
```
Comment (floridamaniac) [inline client/src/Star.jsx:5]: You mentioned this on Slack already, but we'll need some way to get this `score` in order to plug this into other components, but it's non-breaking and everything else looks good so I feel good merging this!

**Review by floridamaniac — APPROVED** [review]

File: client/dist/styles.css [hunk 2]
```diff
@@ -49,3 +49,38 @@
...
+.full-stars {
+  position: absolute;
+  left: 0;
+  top: 0;
+  white-space: nowrap;
+  overflow: hidden;
+  color: #fde16d;
+}
+.empty-stars:before, .full-stars:before {
+  content:"\2605\2605\2605\2605\2605";
```
Comment (ryand8008) [inline client/dist/styles.css:70]: Are the stars an imported image or was it something created in css?

**Review by ryand8008 — APPROVED** [review]
Thanks for checking out the stars! On a quick glance, looks like nothing will break my widget.

File: client/dist/styles.css (hunk 2 above)
Comment (ivyhu630) [reply client/dist/styles.css:70]: It's unicode character for [a star](https://unicode-table.com/en/2605/)

File: client/src/Star.jsx (hunk 1 above)
Comment (ivyhu630) [reply client/src/Star.jsx:5]: Thanks! I will make a ticket so we don't forget about this.

**Pull request merged**